packages/flutter_tools/test/general.shard/build_system/source_test.dart

```diff
@@ -213,22 +213,4 @@ void main() {
     expect(visitor.sources.single.path, r'foo bar.txt');
     expect(visitor.containsNewDepfile, false);
   }));
-
-  test('Non-local engine builds use the engine.version file as an Artifact dependency', () => testbed.run(() {
```

Comment:
This is a testing a caching optimization for builds. This can be restored later.